### ISSUE TITLE
The output of structured logging

### ISSUE THREAD

**Issue by apepkuss** [juntyr/wasi-logger#1]
Hi, I'm using `wasi-logger` in my project. It is really a helpful project. According to my first tries, I found an issues related to the output of structured logging. I simply used the following code to log:

```rust
info!(a = 1; "Hello");
```

The output observed is 

```console
[2024-05-16 11:29:35.794] [wasi_logging_stdout] [info] ***: demo/src/***.rs:161: Hello {Key { key: "a" }: 1}
```
The output of the structured part is different from what I expected. My guess was `"a": 1`. Is it possible to output `"a": 1`? Thanks a lot!

**Comment by juntyr** [comment]
Thanks for reporting this issue!

The fundamental problem is that the  [wasi-logging](https://github.com/WebAssembly/wasi-logging/blob/main/wit/logging.wit) interface doesn’t support structured records itself, so I have to currently format it into the message string.

In the first release I definitely just went with the simplest solution - to just debug format it - but I will try to make it prettier today :)

**Comment by apepkuss** [comment]
Thanks for your reply and looking forward the updates.

**Comment by juntyr** [comment]
Can you try out the output from the version on `main` by switching `wasi-logger` to a git dependency? What do you think? Once you're happy, I'll release a minor version bump.

**Comment by apepkuss** [comment]
It looks great now.

The logging code:

```rust
info!(a=1; "Demo");
```

The output of structured part:

```console
Demo {a: 1}
```

Thank you very much for the quick update!

**Comment by juntyr** [comment]
Great! I've published the update as v0.1.1 to crates.io: https://crates.io/crates/wasi-logger/0.1.1
